app-center-services/src/main/java/org/exoplatform/appcenter/service/ApplicationCenterService.java

```diff
@@ -487,7 +491,7 @@ public ApplicationList getAuthorizedApplicationsList(int offset, int limit, Stri
     resultApplicationsList.setCanAddFavorite(countFavorites < getMaxFavoriteApps());
     resultApplicationsList.setOffset(offset);
     resultApplicationsList.setLimit(limit);
-    resultApplicationsList.setSize(countFavorites);
+    resultApplicationsList.setSize(userApplicationsList.size());
```

Comment:
here we are setting the resultApplicationList size to the current userApplicationsList page size.
In fact, in getApplications, if limit is 10, then userApplicationsList.size will be 10, but the complete list can be larger.

Reply:
Okay